cla/signing-template.md

```diff
@@ -0,0 +1,10 @@
+<yyyy-mm-dd>
+
+I hereby agree to the terms of the "CoatRack <Entity or Individual> Contributor License Agreement", with SHA-256 checksum <sha-256-checksum>.
+
+I furthermore declare that I am authorized and able to make this agreement and sign this declaration.
+
+Signed,
+
+<firstname> <lastname> https://github.com/<github-username>
```

Comment:
I think we should not ask people to put their full name here, the Github username should be sufficient as it clearly maps the contribution to a Github account. 

The personal information like name, address, should be sent to us e.g. via Email instead.

Reply:
Done. The fields 'firstname' and 'lastname' were removed.